src/Three.js

```diff
@@ -156,5 +156,8 @@ export { Curve } from './extras/core/Curve.js';
 export { ShapeUtils } from './extras/ShapeUtils.js';
 export { SceneUtils } from './extras/SceneUtils.js';
 export { WebGLUtils } from './renderers/webgl/WebGLUtils.js';
+export { WEBGL_CONSTANTS } from './renderers/webgl/WebGLConstants.js';
```

Comment:
I think I would keep this as:

```js
export { WebGLConstants } from './renderers/webgl/WebGLConstants.js';
```

Also, I think I would add a `fromGL()` and `toGL()` methods that does the `THREEToWebGL` and WebGLToTHREE` conversions. Just to keep everything in one file.